src/rpc/server.cpp

```diff
@@ -140,44 +138,54 @@ UniValue help(const JSONRPCRequest& jsonRequest)
                     RPCResult::Type::STR, "", "The help text"
                 },
                 RPCExamples{""},
-            }.ToString()
+        [&](const RPCMan& self, const JSONRPCRequest& jsonRequest) -> UniValue
+{
+    self.Check(jsonRequest);
+    if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
+        throw std::runtime_error(
+            self.ToString()
```

Comment:
Since you add a call to `RPCMan::Check`, why then recheck again for `fHelp` flag/`params` size? The if statement could be removed, right?